var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -259,35 +270,43 @@ def initconfig_package_entries(self):
         entries.append("# TPLs")
         entries.append("#------------------{0}\n".format("-" * 60))
 
+        # - BLT
         entries.append(cmake_cache_path("BLT_SOURCE_DIR", spec["blt"].prefix))
+
+        # - RAJA
         if spec.satisfies("+raja"):
             entries.append(cmake_cache_option("{}ENABLE_RAJA_PLUGIN".format(option_prefix), True))
             entries.append(cmake_cache_path("RAJA_DIR", spec["raja"].prefix))
+        else:
+            entries.append(cmake_cache_option("{}ENABLE_RAJA_PLUGIN".format(option_prefix), False))
```

Comment:
Did this fix the CHAI failures you were seeing?

Reply:
Yes